v2/svid/x509svid/svid_test.go

```diff
@@ -0,0 +1,341 @@
+package x509svid_test
+
+import (
+	"errors"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/spiffe/go-spiffe/v2/spiffeid"
+	"github.com/spiffe/go-spiffe/v2/svid/x509svid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+type fileSet struct {
+	certPathPEM string
+	certPathDER string
```

Comment:
Having to maintain both a PEM and DER version of the same data just to ensure the loaded data matches seems like a little too much overhead. Perhaps we can just keep the PEM version and then have a simple function that loads the raw data from a PEM file (doesn't have to be part of pemutil).